packages/pwa-kit-react-sdk/babel.config.js

```diff
@@ -35,7 +35,8 @@ module.exports = {
                 idInterpolationPattern: '[sha512:contenthash:base64:6]',
                 ast: true
             }
-        ]
+        ],
+        ['inline-json-import', {}]
```

Comment:
You'll have to make sure you get 3PP approval for this new package, if you don't remember how to do that we can go over it .